docs/compilation.rst

```diff
@@ -185,46 +187,84 @@ Prerequisites
     ./configure && make
 
 
-Get the DALI source
-+++++++++++++++++++
+Build DALI
+^^^^^^^^^^
+
+1. Get DALI source code:
 
 .. code-block:: bash
 
-  git clone --recursive https://github.com/NVIDIA/dali
-  cd dali
+  git clone --recursive https://github.com/NVIDIA/DALI
+  cd DALI
 
-Make the build directory
-++++++++++++++++++++++++
+2. Create a directory for CMake-generated Makefiles. This will be the directory, that DALI's built in.
 
 .. code-block:: bash
 
   mkdir build
   cd build
 
+3. Run CMake. For additional options you can pass to CMake, refer to :ref:`OptionalCmakeParamsAnchor`.
+
+.. code-block:: bash
 
-Compile DALI
-^^^^^^^^^^^^
+  cmake -D CMAKE_BUILD_TYPE=Release ..
 
-Building DALI without LMDB support:
-+++++++++++++++++++++++++++++++++++
+4. Build. You can pass ``-j`` option to execute it in several threads
```

Comment:
I wouldn't suggest passing `-j` without limits on all system as this can get you OOM during compilation.

Reply:
The line below is ok, I would just phrase it better.